packages/browser/go/jsutil/jsutil_bench_test.go

```diff
@@ -0,0 +1,108 @@
+// +build js,wasm
+
+package jsutil
+
+import (
+	"fmt"
+	"math/cmplx"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/pborman/uuid"
+)
+
+type testValue struct {
+	Uint    uint      `json:"uint"`
+	Uint8   uint8     `json:"uint8"`
+	Uint16  uint16    `json:"uint16"`
+	Uint32  uint32    `json:"uint32"`
+	Uint64  uint64    `json:"uint64"`
+	Int     int       `json:"int"`
+	Int8    int8      `json:"int8"`
+	Int16   int16     `json:"int16"`
+	Int32   int32     `json:"int32"`
+	Int64   int64     `json:"int64"`
+	Float32 float32   `json:"float32"`
+	Float64 float64   `json:"float64"`
+	Byte    byte      `json:"byte"`
+	Rune    rune      `json:"rune"`
+	String  string    `json:"string"`
+	Bool    bool      `json:"bool"`
+	Time    time.Time `json:"time"`
+}
+
+func newTestValue() *testValue {
```

Comment:
Hypothetically, these test values could skew the benchmark if the savings are considerably different for small number types compared to things like large strings. I think that it would be good to benchmark against arrays of asset data, big ints, or a mix of types that better describe our use case (like `zeroex.SignedOrder`). 

This is not to say that this benchmark is not useful. I just think that it would be good to have one more that is more tailored to our use-case.

Reply:
There's no single type that we could use here that would cover all the places we use `InefficientlyConvertToJS` throughout the codebase. The idea here is just to give us a decent idea of how `InefficientlyConvertToJS` performs generally. Based on what I have seen so far, I think it is extremely unlikely that this change will make `InefficientlyConvertToJS` _slower_, no mater what type we are converting.

 If we have a need to optimize the encoding/conversion of a specific type (e.g. in the `database` package), we should just use a separate benchmark that looks at that type specifically.

Reply:
I also think that’s unlikely. I think the real point is that we should have some benchmarks on more realistic benchmarks to see how much of a performance boost we get for Mesh specific use cases.